src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.ValueGenerator/Main.cs

```diff
@@ -150,13 +197,44 @@ private Result GetResult(IComputeRequest request)
             };
         }
 
+        private List<Result> GetSuggestionFuzzyResults(Query query, List<Result> results)
+        {
+            foreach (var generatorData in QueryHelper.GeneratorDataList)
+            {
+                var matchScore = StringMatcher.FuzzySearch(query.Search, generatorData.Keyword).Score;
+
+                if (matchScore > 0)
+                {
+                    results.Add(new Result
+                    {
+                        Title = QueryHelper.GetResultTitle(generatorData),
+                        SubTitle = QueryHelper.GetResultSubtitle(generatorData),
+                        IcoPath = GetIcoPath(),
+                        Score = matchScore,
+                        ToolTipData = new ToolTipData(QueryHelper.GetResultTitle(generatorData), QueryHelper.GetResultSubtitle(generatorData)),
+                        ToolTipVisibility = Visibility.Visible,
+                        QueryTextDisplay = generatorData.Keyword,
```

Comment:
Don't we need both: Action keyword and generator keyword?

Reply:
see: https://github.com/microsoft/PowerToys/pull/33490#discussion_r1659516403